web/src/main/java/com/stezhka/web/util/JsonJacksonConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.stezhka.web.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.stezhka.web.dto.MovieDto;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class JsonJacksonConverter {
+
+    @Autowired
+    private ModelMapper modelMapper;
```

Comment:
ModelMapper does not used. Can be removed from the class

Reply:
Fixed